src/Rector/Deprecation/PathAliasRepositoryRector.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Renames deprecated Drupal::services('path.alias_manager') argument.
```

Comment:
Should this say `path.alias_repository`?

Reply:
Yes, good catch.